pkg/controllers/vmsetcontroller/vmsetcontroller.go

```diff
@@ -345,22 +344,28 @@ func (v *VirtualMachineSetController) reconcileVirtualMachineSet(vmset *hfv1.Vir
 
 			_, err = v.hfClientSet.HobbyfarmV1().VirtualMachines(util.GetReleaseNamespace()).UpdateStatus(v.ctx, vm, metav1.UpdateOptions{})
 
-
-
 			if err != nil {
 				glog.Error(err)
 			}
 
-
 			err = util.VerifyVM(v.vmLister, vm)
 			if err != nil {
 				glog.Error(err)
 			}
 		}
-
-		//TODO handle case of scaling down VMSets
 	}
-
+//-----------------------handle case of scaling down VMSets
```

Comment:
this is of you old PR. Next time start of with the main branch of the project :P